
float32[18] joint_angles
float32[18] foot_position